dapp/src/components/name-card/NameCard.tsx

```diff
@@ -30,11 +32,12 @@ export function NameCard({
                 nftDisplayVariants({ size }),
             )}
         >
-            <AvatarDisplay
+            <NameCardDisplay
                 name={name}
                 size={size}
                 badge={badge}
                 button={menuOptions ? <ContextMenuButton options={menuOptions} /> : null}
+                fallbackUrl={auctionDisplay}
```

Comment:
if we dont use `staticImageSrc` then the component will go through the avatar route and execute unnecessary hooks
```suggestion
                staticImageSrc={auctionDisplay}
```